src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/BulkPrintStaffNotificationServiceTest.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.cmc.claimstore.MockSpringTest;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.reform.sendletter.api.Document;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class BulkPrintStaffNotificationServiceTest extends MockSpringTest {
+
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+
+    @Autowired
+    private BulkPrintStaffNotificationService service;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    @Autowired
+    private StaffEmailProperties emailProperties;
+
+    private List<Document> documents;
+
+    private Claim claim;
+
+    @Before
+    public void setup() {
+        documents = new ArrayList<>();
+        documents.add(new Document("template1", new HashMap<>()));
+        documents.add(new Document("template2", new HashMap<>()));
+
+        claim = SampleClaim
+            .builder()
+            .build();
+
+        when(pdfServiceClient.generateFromHtml(any(byte[].class), anyMap()))
+            .thenReturn(PDF_CONTENT);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldThrowNullPointerWhenGivenNullClaim() {
+        service.notifyFailedBulkPrint(null, null);
```

Comment:
There are 2 `null` values passed in here but the test mentions claim only. Please provide a value for the other one and an additional test to check that NPE is thrown when the other one is `null`.